service/src/test/java/uk/gov/hmcts/reform/fpl/service/StandardDirectionsServiceTest.java

```diff
@@ -91,6 +93,31 @@ void shouldReturnFalseWhenThereAreNoEmptyDates() {
         assertThat(service.hasEmptyDates(caseData)).isFalse();
     }
 
+    @Test
+    void shouldPopulateStandardDirections() {
+        LocalDate hearingDate = LocalDate.of(2099, 6, 1);
```

Comment:
maybe:
LocalDate hearingDate = LocalDate.now().plusYears(10)
to have always future date